src/timeline/MetaAnimation.js

```diff
@@ -244,6 +244,29 @@ FORGE.MetaAnimation.prototype.destroy = function()
 };
 
 /**
+ * Accessors to timeline
+ * @name FORGE.MetaAnimation#timeline
```

Comment:
Running is not used internally it is just a property that I wanted to expose in the API.